test/relations.test.js

```diff
@@ -2421,9 +2572,18 @@ describe('relations', function() {
       Author.findById(author.id, function(err, author) {
         author.pictures(function(err, pics) {
           pics.should.have.length(3);
-          pics[0].name.should.equal('Author Pic 1');
-          pics[1].name.should.equal('Author Pic 2');
-          pics[2].name.should.equal('Example');
+
+          if (isCassandraConnector) {
```

Comment:
Please rewrite using `containDeep`, similarly to what I have shown in my comments above.